src/main/java/sirius/biz/codelists/CodeListImportJob.java

```diff
@@ -0,0 +1,158 @@
+        this.languageParameter = languageParameter;
+        languageParameter.ifPresent(lang -> {
+            this.languagePrefix = lang + "_";
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_VALUE)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "CodeListEntryData.value",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_VALUE));
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_ADDITIONAL_VALUE)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "CodeListEntryData.additionalValue",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_ADDITIONAL_VALUE));
+            this.dictionary.addField(FieldDefinition.stringField(languagePrefix + CLE_DESCRIPTION)
+                                                    .withLabel(languagePrefix.toUpperCase() + NLS.get(
+                                                            "Model.description",
+                                                            lang))
+                                                    .addAlias(languagePrefix + CLE_DESCRIPTION));
+        });
+    }
+
+    @Override
+    protected E fillAndVerify(E entity, Context context) {
+        languageParameter.ifPresent(lang -> {
+            if (context.keySet().stream().noneMatch(key -> key.startsWith(languagePrefix))) {
+                throw Exceptions.createHandled().withNLSKey("Translations.importLanguageNotFound").handle();
+            }
+        });
+        return super.fillAndVerify(entity, context);
+    }
+
+    @Override
+    protected E findAndLoad(Context data) {
+        data.set(CodeListEntry.CODE_LIST.getName(), codeList);
+        return super.findAndLoad(data);
+    }
+
+    @Override
+    public void execute() throws Exception {
+        process.addLink(new ProcessLink().withLabel("$CodeList").withUri("/code-list/" + codeList.getIdAsString()));
+        process.updateTitle(Strings.apply("%s - %s", process.getTitle(), codeList.getCodeListData().getName()));
+        super.execute();
+    }
+
+    /**
+     * Updates {@link sirius.biz.translations.Translation Translations} for the {@link CodeListEntry}, if a language is
+     * selected for the import.
+     *
+     * @param entity  the entity to persist
+     * @param context the row represented as context
+     */
+    @Override
+    protected void createOrUpdate(E entity, Context context) {
+        super.createOrUpdate(entity, context);
+
+        // update the translation texts for the selected language
+        languageParameter.ifPresent(lang -> {
```

Comment:
importhandler?

Reply:
If we move it to the handler, we would have to duplicate the code for SQL and Mongo. That would be unnecessary, since there are no database dependent instructions needed to update translation texts. The Translation entity handles that by itself

Reply:
To prevent code duplication, it has been agreed to keep the code in this class.